ovirt-engine-hyper-upgrade.py

```diff
@@ -224,13 +224,15 @@ def main():
     parser.add_argument(
         '--check-upgrade-rhv-4-0',
         action='store_true',
-        help='Check if RHV 4.0 channels have upgrade available',
+        help='Check if RHV 4.0 channels have upgrade available. '
+             'Also enable 4.1 channels',
     )
 
     parser.add_argument(
         '--check-upgrade-rhv-4-1',
         action='store_true',
-        help='Check if RHV 4.1 channels have upgrade available',
+        help='Check if RHV 4.1 channels have zstream upgrade available. '
+             'Also enable 4.2 channels',
```

Comment:
Are the texts for these two options intentionally different?

I am not sure we need them. Perhaps better to have a single option "--upgrade-to-version=<version>", default to 4.2 currently. Not sure I understand the intention, though.

Reply:
I am not sure about if we intend to support hosts in the future, @didib  @sandrobonazzola might know.